### DISCUSSION TITLE
Deploying Strapi using Amazon Lightsail instance

### DISCUSSION THREAD

**Discussion by XBShu** [strapi/strapi#8681] [Legacy Topics]
I'm trying to deploy Strapi using an Amazon Lightsail isntance. I have installed Node and cloned the strapi repository from Github, but running npm start in the VPS still tells me to go to localhost:1337. How do I set it up so Strapi can be accessed through my instance's IP address?

EDIT: fixed the issue! I originally wasn't able to access the app through my instance IP address due to the default firewall permissions. Everything worked after changing them in the networking tab.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
